src/core.js

```diff
@@ -1206,6 +1206,7 @@ define([
         }).on('remove-question', function (e) {
             if (!e.isInternal) {
                 var prev = _this.jstree("get_prev_dom", e.mug.ufid);
+                _this.showVisualValidation(null);
```

Comment:
On further investigation it looks like [the bug](http://manage.dimagi.com/default.asp?75697) only appears when deleting the last question on the form.